features/graph/rest/src/main/java/org/opennms/netmgt/graph/rest/api/Query.java

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2019-2019 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2019 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.netmgt.graph.rest.api;
+
+import java.util.List;
+
+public class Query {
+
+    public static final int DEFAULT_SEMANTIC_ZOOM_LEVEL = 1;
+
+    private Integer szl;
+
+    private List<String> verticesInFocus;
+
+    public Integer getSzl() {
```

Comment:
I wonder if we shouldn't call the method getSemanticZoomLevel()? I think this makes the code more readable than the abbreviation szl. (there are multiple occurrences in the code of "szl")

Reply:
Or at least "zoom" or "zoomLevel" instead, that would make more sense.

Reply:
I decided to name it `semanticZoomLevel`. Even if the statement `SZL` or `szl` is a common term in context of Topology/Graph Service.